tests/test-demo-buildsystem.sh

```diff
@@ -19,9 +19,12 @@
 
 set -euo pipefail
 
-if ! fusermount --version >/dev/null 2>&1; then
+if ! which fusermount >/dev/null 2>&1; then
     echo "1..0 # SKIP no fusermount"
     exit 0
+elif ! capsh --print | grep ^Current | grep -q cap_sys_admin; then
```

Comment:
Oh right, this will turn off the tests when run inside a desktop system.  It's not securebits I think so much as the bounding set.  Changing "Current" to "Bounding set" should work in both cases I think.